transmart-rest-api/grails-app/controllers/org/transmartproject/rest/PatientSetController.groovy

```diff
@@ -85,4 +87,23 @@ class PatientSetController {
         ),
         [status: 201]
     }
+
+    /**
+     * Disable created patient set.
+     *
+     * POST /patient_sets/<result_instance_id>
+     */
+    def disable(Long id) {
+        respond queriesResource.runDisablingQuery(id, currentUser.username),
+                [status: 201]
+    }
+
+    private wrapPatients(Object source) {
```

Comment:
This is supposed to be a function for wrapping `QueryResultSummary` objects, right?
With `componentType: QueryResultSummary` etc...